vcbuild.bat

```diff
@@ -81,7 +84,6 @@ goto next-arg
 
 :args-done
 if defined upload goto upload
-if defined jslint goto jslint
```

Comment:
Good catch. This change enables building and testing, when jslint is specified. But it has the side effect of not getting to jslint unless you run tests. Will fix.